.travis.yml

```diff
@@ -1,19 +0,0 @@
-language: node_js
-os:
-  - linux
-  - osx
-addons:
-  ssh_known_hosts:
-    - github.com
-node_js:
-  - 10.22.1
-  - 12.18.4
-
-cache:
-  directories:
-    - node_modules
-install:
-  - npm install
-script:
-  - npm run lint
```

Comment:
@bookernath @junedkazi We missed the linting step. 
Was it intentional or accidental?
If accidental I can create a followup PR adding this missing step

Reply:
yes please. Can you please open a PR to add it.